wolfcrypt/src/wc_port.c

```diff
@@ -1076,6 +1081,48 @@ size_t wc_strlcat(char *dst, const char *src, size_t dstSize)
 }
 #endif /* USE_WOLF_STRLCAT */
 
+#ifdef USE_WOLF_STRCASECMP
+int wc_strcasecmp(const char *s1, const char *s2)
+{
+    char c1, c2;
+    for (;
```

Comment:
Why is for statement over multiple lines. We never do this normally.
Where did this code come from?

Reply:
typographically matches the code in `wc_strncasecmp()` below, and I wanted to visually highlight that it's an empty clause.  I can one-line them of course -- sounds like you want?

the code came from me on Friday, black box clean room.

Reply:
Please conform to our normal format.

Reply:
:heavy_check_mark: